tests/serialization/test_deserialization.py

```diff
@@ -0,0 +1,78 @@
+"""Deserialization tests.
```

Comment:
given that these tests are already largely covered by alternative creation and normal serialization tests, is this necessary? the invaldi test also seems fitting for the alternative creation test file and doesnt need its own file

Reply:
hmm if you insist, some of the tests still duplicate tests that are done in alternative creation for acqf functions

Reply:
Ok, because this PR seems never ending, let me suggest the following: we keep it for now as a double safety net (they are super cheap) and then the cleanup will automatically happen once we are done with the remaining hypothesis parts, at which point an complete revisit of the test cases will be overdue anyway (much of the stuff will have become obsolete by then). Right now, I think they are necessary, but I really don't want to make this PR any longer anymore :D